### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by drazioti**
The following changes have been made.
1. Rename 'genume' to 'about' and adding some more info
2. If you don't have a cd in your laptop/deskotp awk provides an error, I changed the file cd.sh
3. I added a screenshot in README
4. I commented line 124 in genume/genume/view.py, since for some reason refresh button did not work for me. Of course this must be handled.

### CODE REVIEW

**Comment by MaanooAk** [comment]
I am requesting a review from @Sima214 and after that I'm merging (@drazioti if the pr is not ready yet, please add a WIP in the title).

After, I suggest to change the screenshot image (yours have two X buttons).

Also, an other issue is shown, by renaming the `genume` folder to script to `about`, it is now the first on list. We should somehow have a way to order the list.

**Comment by drazioti** [comment]
Yes and I won't add anything for this pr. 

Yes, it has two X buttons. I can not do anything about it. This is something from gtk I assume. Pls, add your screenshot.

About the renaming, I think we can not order the buttons (at least I don't know how), the program order them alphabetically.

**Comment by MaanooAk** [comment]
@drazioti about the sorting, @Sima214 has some ideas to improve the protocol (between genume and scripts), I am just stating one more _need_ for the protocol.

**Comment by drazioti** [comment]
Agreed. It would be a nice feature to manually order the buttons.

**Comment by Sima214** [comment]
I won't be able to look at it until friday.

File: scripts/software/apps.sh [hunk 1]
```diff
@@ -10,8 +10,7 @@ function check {
 	key="app_$name"
 
 	if command -v "$name" > /dev/null; then
-
-		echo VALUE BAS "$key" \"$($name $version 2>&1 | head -n1)\" GROUP "apps"
+		echo VALUE BAS "$key" \"$($name $version 2>&1 | sed '/./q; d')\" GROUP "apps"
```
Comment (MaanooAk) [inline scripts/software/apps.sh]: @drazioti As I understand it, you changed `head -n1` to `-n2` to support some programs with version outputs that start with an empty line. But in other cases, the second lines contain copyright and other unnecessary stuff.
So I changed back to get only the first line but skip the empty lines.

If this was not the reason, please correct me.

Comment (MaanooAk) [reply scripts/software/apps.sh]: Now, after my change (`sed '/./q; d'`),  it works, this is my output:

...
app_perl: This is perl 5, version 22, subversion 1 (v5.22.1) built for x86_64-linux-gnu-thread-multi
app_nmap: Nmap version 7.01 ( https://nmap.org )**
app_nano: GNU nano, version 2.5.3
...

File: genume/view/main.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def set_button_location(s, b, allocation):
 
         refresh_button = self.generate_refresh_button()
         main_view.add_overlay(refresh_button)
-        main_view.set_overlay_pass_through(refresh_button, True)
+        # main_view.set_overlay_pass_through(refresh_button, True)
```
Comment (MaanooAk) [inline genume/view/main.py]: This based on the doc should break the app by making everything but the refresh button clickable, but when I run it everything works.

We should wait until #62 is closed first (I will look into it in the next days)

**Pull request merged**